dailystats.py

```diff
@@ -19,9 +19,12 @@
 
 
 def read_stats_file(csv_file_name):
-    """ Reads the daily gov.uk covid 19 csv file and returns a dictionary where:
-            key fields are the countries/regions
-            values are a list of dates and daily case counts  """
+    """Parse a CSV file in the daily gov.uk covid-19 format.
+
+    :param csv_file_name: The path of the csv file to parse.
+    :return: A dictionary mapping countries/regions to a list of dates and daily
+        case counts.

```

Comment:
This is the standard way of documenting arguments and return values - I think pycharm has in-built support for it